spring-cloud-gcp-pubsub-stream-binder/src/test/java/org/springframework/cloud/gcp/stream/binder/pubsub/properties/PubSubExtendedBindingsPropertiesTests.java

```diff
@@ -36,14 +37,8 @@ public void testExtendedProperties() {
 		BinderFactory binderFactory = context.getBeanFactory().getBean(BinderFactory.class);
 		PubSubMessageChannelBinder binder = (PubSubMessageChannelBinder) binderFactory.getBinder("pubsub", MessageChannel.class);
 
-		// Print out the default values; they should both be false.
-		System.out.println("default for INPUT: "
-				+ binder.getExtendedConsumerProperties("input").isAutoCreateResources());
-		System.out.println("default for CUSTOM INPUT: "
-				+ binder.getExtendedConsumerProperties("custom-in").isAutoCreateResources());
-
-		assertThat(binder.getExtendedConsumerProperties("input").isAutoCreateResources()).isFalse();
 		assertThat(binder.getExtendedConsumerProperties("custom-in").isAutoCreateResources()).isFalse();
+		assertThat(binder.getExtendedConsumerProperties("input").isAutoCreateResources()).isTrue();
```

Comment:
Wait. Sounds like everything works as expected now.

What are we missing then? 🤷‍♂️

Reply:
Ah not quite, I've modified the test so it can reproduce the problem, however it is still broken.